solr/core/src/java/org/apache/solr/servlet/HttpSolrCall.java

```diff
@@ -354,13 +352,15 @@ protected void init() throws Exception {
    * recursively try this all one more time while ensuring the alias and collection info is sync'ed
    * from ZK.
    */
-  protected DocCollection resolveDocCollection(List<String> collectionsList) {
+  protected DocCollection resolveDocCollection(String collectionName) {
```

Comment:
Its javadocs appear outdated now.

I like the signature change.

Reply:
Updated.